teams.json

```diff
@@ -0,0 +1,36 @@
+{
+  "metamask/accounts-controller": "team-accounts",
+  "metamask/address-book-controller": "team-confirmations",
+  "metamask/announcement-controller": "team-wallet-ux",
+  "metamask/approval-controller": "team-confirmations",
+  "metamask/assets-controllers": "team-assets",
+  "metamask/base-controller": "team-wallet-framework",
+  "metamask/build-utils": "team-wallet-framework",
+  "metamask/chain-controller": "team-accounts",
+  "metamask/composable-controller": "team-wallet-framework",
+  "metamask/controller-utils": "team-wallet-framework",
+  "metamask/ens-controller": "team-confirmations",
+  "metamask/eth-json-rpc-provider": "team-wallet-api-platform",
+  "metamask/gas-fee-controller": "team-confirmations",
+  "metamask/json-rpc-engine": "team-wallet-api-platform",
+  "metamask/json-rpc-middleware-stream": "team-wallet-api-platform",
+  "metamask/keyring-controller": "team-accounts",
+  "metamask/logging-controller": "team-confirmations",
+  "metamask/message-manager": "team-confirmations",
+  "metamask/name-controller": "team-confirmations",
+  "metamask/network-controller": "team-wallet-framework",
+  "metamask/notification-controller": "team-snaps-platform",
+  "metamask/notification-services-controller": "team-notifications",
+  "metamask/permission-controller": "team-wallet-api-platform",
+  "metamask/permission-log-controller": "team-wallet-api-platform",
+  "metamask/phishing-controller": "",
```

Comment:
I wans't able to find `@MetaMask/product-safety` team label. @Gudahtt Could you provide it and double check if rest of mapping is correct?

Reply:
It's `team-product-safety`. They were missing from the JSON file I sent you (they were just added to that today)